### DISCUSSION TITLE
Sample of how signup and login is handled

### DISCUSSION THREAD

**Discussion by Anan7Codes** [cuongdevjs/reactjs-social-login#143] [Q&A] [answered]
Hey guys, is there any repo on how signup and signin are handled securely on the server side?

**Comment by cuongdevjs** [comment, marked as answer]
there are many samples on the internet about how to handle accessToken on the server side, you can refer to: https://codeculturepro.medium.com/implementing-authentication-in-nodejs-app-using-oauth2-0-59fee8f63798

**Reply by Anan7Codes** [reply]
Cool thanks!
